api_docs/unmerged.d/ZF-8d0c68

```diff
@@ -4,3 +4,6 @@
 * [`PATCH /invites/multiuse/{invite_id}`](/api/edit-invite-link):
   Added support for editing `invite_as` and `stream_ids` fields of
   reusable invitation links.
+* [`PATCH /invites/multiuse/{invite_id}`](/api/edit-invite-link):
```

Comment:
We don't usually describe which users have permissions in the changelog, so this can be removed,